client/app/dedicated/server/interfaces/rename/interfaces-rename.routing.js

```diff
@@ -0,0 +1,19 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicated.server.interfaces.rename', {
+    url: '/rename',
+    views: {
+      modal: {
+        component: 'dedicatedServerInterfacesRename',
+      },
+    },
+    params: {
+      interface: null,
+    },
+    layout: 'modal',
+    translations: { value: ['.'], format: 'json' },
+    resolve: {
+      goBack: /* @ngInject */ $state => () => $state.go('^'),
+      interface: /* @ngInject */ $stateParams => _.cloneDeep($stateParams.interface),
```

Comment:
Why are you deep clone the `interface` parameter ?

```suggestion
      interface: /* @ngInject */ $transition$ => $transition$.params().interface,
```

Reply:
From what I was said, It's to avoid the value of `interface` to change outside of the modal (here with the rename). Using `$transition$` will do this too ? Why using `$transition$` instead of `$stateParams` ?

Reply:
`$stateParams` is deprecated 😉

Reply:
I'm working on how we handle code reviews right now, so I take your remark into account (and I fully agree with it)